CONTRIBUTING.md

```diff
@@ -182,6 +182,98 @@ components/
     my-component.spec.js
 ```
 
+#### Spec File Setup
+
+Each component will require a DOM-like environment in order to be queried for the custom element and, thus assert on the component details.
+
+The following is an example of a spec file, with imports, a global mock (`fetch`), and the `describe`/`it` blocks.
+
+```js
+import { expect } from "@esm-bundle/chai";
+import "./my-custom-element.js";
+
+window.fetch = function () {
```

Comment:
Not sure if you saw my [comment](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/issues/110#issuecomment-2411605929) but it shouldn't be required that all tests need this, just those take advantage of Greenwood's Content as Data APIs.

I think it's worth calling that out but would just want to at least caveat that just to save contributors a little unnecessary boilerplate.